ts/Accessibility/Components/LegendComponent.ts

```diff
@@ -682,10 +682,9 @@ namespace LegendComponent {
         const items = this.legend.allItems;
         const oldIx = this.accessibility &&
                 this.accessibility.components.legend.highlightedLegendItemIx;
-        const itemToHighlight = items[ix],
-            legendItem = itemToHighlight.legendItem || {};
-
-        if (itemToHighlight) {
+        const itemToHighlight = items[ix], legendItem = itemToHighlight?.legendItem || {};;
+        
+        if (itemToHighlight && legendItem) {
```

Comment:
If you are setting the `legendItem` to be equal to `{}`, then this value will always be true, so this check is redundant.

Reply:
I removed the assignment to `{}`. If you think it would be better to keep it, and remove the check in line 687 let me know.